src/components/PostInfo/PostInfo.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { CommentList } from '../CommentList/CommentList';
+import { UserInfo } from '../UserInfo/UserInfo';
+import { PreparedPost } from '../../Types/Types';
+import './PostInfo.scss';
+
+type Props = {
+  onePost: PreparedPost;
+};
+
+export const PostInfo:React.FC<Props> = ({ onePost }) => (
+  <div className="post">
+    <div className="post__head">
+      <h2 className="post__title">
+        {onePost.title}
+      </h2>
+      <div className="post__user">
+        {onePost.user ? (
+          <UserInfo user={onePost.user} />
+        ) : ('')}
+      </div>
```

Comment:
use conditional rendering with && or ||,
and you don't need this div at all if user is null